.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/administrate
+    docker:
+      - image: circleci/ruby:2.4.1-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: administrate
+          RAILS_ENV: test
+      - image: postgres:9.5-alpine
+        environment:
+          POSTGRES_USER: administrate
+          POSTGRES_DB: administrate-prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore Cached Dependencies
+      - type: cache-restore
+        name: Restore bundle cache
+        key: administrate-{{ checksum "Gemfile.lock" }}
+
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+
+      # Install Appraisal gemfiles
+      - run: bundle exec appraisal install
+
+      # Cache Dependencies
+      - type: cache-save
+        name: Store bundle cache
+        key: houndci-{{ checksum "Gemfile.lock" }}
+        paths:
+          - vendor/bundle
+
+      # Wait for DB
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+
+      # Setup the environment
+      - run: cp .sample.env .env
+
+      # Setup the database
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
```

Comment:
Could these two lines be replaced with one?

`bundle exec rake db:create db:schema:load`

Reply:
Or alternatively: `bundle exec rake db:setup` ?